test/node/connection-callback-multiple-calls.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const tap = require('tap');
+const jstp = require('../..');
+
+const app = new jstp.Application('app', {});
+const server = jstp.net.createServer({ applications: [app] });
+
+tap.plan(1);
+
+server.listen(0, () => {
+  const port = server.address().port;
+  jstp.net.connect(app.name, null, port, (error, connection) => {
+    tap.pass('must call callback once');
+    if (error) return;
```

Comment:
I'd prefer this to use assertion here. But if you like it his way - you can go ahead and merge.